src/modelmanager.cpp

```diff
@@ -167,6 +175,7 @@ void processPipelineConfig(rapidjson::Document& configJson, const rapidjson::Val
 
     std::vector<NodeInfo> info{
         {NodeKind::ENTRY, "request"}};
+    processPipelineInputs(pipelineConfig.FindMember("inputs"), "request", info[0].outputNameAliases);
```

Comment:
I think that maybe it would be good to have that name explicit as a static PipelineDefinition string?

Reply:
I mean the names of entry/exit nodes

Reply:
Done